web-app/js/angular/controllers/OpusController.js

```diff
@@ -283,6 +284,62 @@ profileEditor.controller('OpusController', function (profileService, util, messa
         form.$setDirty();
     };
 
+    self.addBioStatusList = function () {
+        if(self.canAddBioStatusList()){
```

Comment:
Don't bother with this - since you've coded everything else to handle multiple lists, just let them select as many as they want.